addons/mail/views/discuss_call_settings_views.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<odoo>
+    <data>
+        <record id="mail.discuss_call_settings_action" model="ir.actions.client">
```

Comment:
:socks: Actually it should be in a ``data/ir_actions_client.xml`` file :) 

I know it is debatable, but "views" generally contains model-related views + template for frontend, not "stuff used in views". Purpose is to be able to find low-level data records in data, like ir_cron, ir_actions_server, ... just looking at module structure.